src/App.js

```diff
@@ -9,7 +11,14 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <>
+    <h1 className="app-title">React Article</h1>
```

Comment:
Page should usually have only one `h1` element. So, this header should be deleted.